account_check_printing_report_base/report/check_print.py

```diff
@@ -113,3 +113,8 @@ def _get_report_values(self, docids, data=None):
             '_format_date_to_partner_lang': self._format_date_to_partner_lang,
         }
         return docargs
+
+
+class ReportCheckPrintA4(models.AbstractModel):
```

Comment:
Why do you need an extra model?

Reply:
If I don't add this, when I try to print on the report, I get an error because "paid_lines" is NoneType

Reply:
I'm not fully understanding this, but let's unblock it.